nflow-rest-api-jax-rs/src/main/java/io/nflow/rest/v1/jaxrs/WorkflowInstanceResource.java

```diff
@@ -116,12 +109,10 @@ public Response createWorkflowInstance(
     @ApiResponse(code = 409, message = "If workflow was executing and no update was done") })
   public Response updateWorkflowInstance(@ApiParam("Internal id for workflow instance") @PathParam("id") long id,
       @ApiParam("Submitted workflow instance information") UpdateWorkflowInstanceRequest req) {
-    try {
+    return handleExceptions(() -> {
```

Comment:
Can this throw EmptyResultDataAccessException if the workflow is not found?
-> Your exception mapper will then fail to format the string due to missing argument?

Reply:
It may, but not always. Anyway the exception mapper does not fail on this anymore.